tools/varstore-sb-state.c

```diff
@@ -110,8 +108,10 @@ int main(int argc, char **argv)
     printf("Removing dbx...\n");
     do_rm(&gEfiImageSecurityDatabaseGuid, "dbx");
 
-    if (!strcmp(argv[optind + 1], "user"))
+    if (!strcmp(argv[optind + 1], "user")) {
+        load_auth_data();
```

Comment:
This won't work when using the chroot option since the auth data needs to be loaded from the filesystem before `drop_privileges()` is called.

You could just leave `load_auth_data()` where it is and guard it with `if (!strcmp(argv[optind + 1], "user"))`.

Reply:
Good catch Ross, thanks.  I just reverted this and pushed up the corrected the change.